csvcubed/csvcubed/schema/cube-config/v1_0/schema.json

```diff
@@ -26,25 +26,29 @@
         "keywords": {"$ref": "#/definitions/v1.0/keywordTypes"},
         "dataset_issued": {
             "description": "Date time that the data set was initially published in ISO 8601 format, e.g. 2022-03-31 or 2022-03-31T12:54:30Z",
-            "type": "string",
-            "format": "date-time"
+            "type": "string"
         },
         "dataset_modified": {
             "description": "Date time that the data set was last modified published in ISO 8601 format, e.g. 2022-03-31 or 2022-03-31T12:54:30Z",
-            "type": "string",
-            "format": "date-time"
+            "type": "string"
```

Comment:
You'll notice I've left the date-time fields as just "string". We parse the values as ISO-8601 date/date-times, however the JSON schema specification says that the `date-time` format requires [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) formatted date-times which differ slightly.